index.css

```diff
@@ -0,0 +1,48 @@
+
+.category {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
+    width: 15rem;
+    height: 15rem;
+    border-radius: 5px;
+    background-color: lightgray;
+    cursor: pointer;
+
+}
+.category-cards {
+    display: flex;
+    flex-flow: row wrap;
+    justify-content: center;
+    align-items: center;
+    gap:5rem;
+}
+.cat-image {
+    width: 168px;
+    font-size: 2rem;
+    height: 117px;
+}
+.cat-item {
+    font-size: 1.5rem;
+    text-decoration: none;
+}
+.home-logo {
+    width: 45rem;
+    height: 40rem;
+}
+.new-arrivals {
+    width: 25rem;
+    height: 20rem;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: lightgray;
```

Comment:
You can put all the colors in the root variables

Reply:
sure